src/themes/five/src/components/OrderDetails/index.js

```diff
@@ -210,6 +216,21 @@ const OrderDetailsUI = (props) => {
     return order?.taxes?.filter(tax => tax?.type === 1)?.reduce((carry, tax) => carry + (tax?.summary?.tax_after_discount ?? tax?.summary?.tax), 0)
   }
 
+  const handleReorder = async (orderId) => {
```

Comment:
Add this function in functional component from ordering-components, in order to reuse the function in another apps

Reply:
move function to ordering component
https://github.com/Ordering-Inc/ordering-components/pull/681